core/src/trezor/ui/layouts/tt/__init__.py

```diff
@@ -1150,32 +1153,80 @@ async def confirm_sign_identity(
 
 
 async def confirm_signverify(
-    coin: str, message: str, address: str, verify: bool
+    message: str,
+    address: str,
+    verify: bool,
+    path: str | None = None,
+    account: str | None = None,
+    chunkify: bool = False,
 ) -> None:
     if verify:
-        title = f"VERIFY {coin} MESSAGE"
+        address_title = "VERIFY ADDRESS"
         br_type = "verify_message"
     else:
-        title = f"SIGN {coin} MESSAGE"
+        address_title = "SIGNING ADDRESS"
         br_type = "sign_message"
 
-    await confirm_blob(
-        br_type,
-        title,
-        address,
-        "Confirm address:",
-        br_code=BR_TYPE_OTHER,
+    address_layout = RustLayout(
+        trezorui2.confirm_address(
+            title=address_title,
+            data=address,
+            description="",
+            verb="CONTINUE",
+            extra=None,
+            chunkify=chunkify,
+        )
     )
 
-    await confirm_blob(
-        br_type,
-        title,
-        message,
-        "Confirm message:",
-        hold=not verify,
-        br_code=BR_TYPE_OTHER,
+    items = []
```

Comment:
Could be worth to type this ... `items: list[tuple[str, str]]`, so we have an extra check

Reply:
0165c99fa6e8204325e576ac3d3b9da13f04dba7